<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpsLogs</class>
 <widget class="QWidget" name="OpsLogs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>309</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../data/kstars.qrc">:/icons/AlignWarning.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable Logs to diagnose issues with Ekos and INDI. Only enable the necessary logs to help in diagnosing specific problems. Logs consume resources and may slow the system down. Turn off logs when no longer used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Logs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Output:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5" colspan="2">
       <widget class="QCheckBox" name="kcfg_GuideLogging">
        <property name="toolTip">
         <string>Log Ekos Guide module activity</string>
        </property>
        <property name="text">
         <string>Guide</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modulesGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="11" column="9">
       <widget class="QCheckBox" name="kcfg_INDIAuxiliaryLogging">
        <property name="text">
         <string>Auxiliary</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="7" column="2" colspan="3">
       <widget class="QCheckBox" name="kcfg_AlignmentLogging">
        <property name="toolTip">
         <string>Log Ekos Alignment module activity</string>
        </property>
        <property name="text">
         <string>Alignment</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modulesGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="6" colspan="2">
       <widget class="QRadioButton" name="kcfg_LogToFile">
        <property name="toolTip">
         <string>Log output to log file</string>
        </property>
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item row="7" column="7" colspan="2">
       <widget class="QCheckBox" name="kcfg_MountLogging">
        <property name="toolTip">
         <string>Log Ekos Mount module activity</string>
        </property>
        <property name="text">
         <string>Mount</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modulesGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="1" rowspan="5">
       <widget class="QLabel" name="label_6">
        <property name="toolTip">
         <string>Have problem with Ekos? Turn on logging for Ekos modules that exhibit issues.</string>
        </property>
        <property name="text">
         <string>Ekos</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QRadioButton" name="kcfg_LogToDefault">
        <property name="toolTip">
         <string>Log debug messages to default output device used by the platform (e.g. Standard Error)</string>
        </property>
        <property name="text">
         <string>Defaul&amp;t</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="9" colspan="3">
       <widget class="QCheckBox" name="kcfg_INDIFilterWheelLogging">
        <property name="text">
         <string>Filter Wheel</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="8" column="0" colspan="13">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8" colspan="3">
       <widget class="QRadioButton" name="kcfg_DisableLogging">
        <property name="toolTip">
         <string>Disable all logging output</string>
        </property>
        <property name="text">
         <string>&amp;Disable</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">verbosityButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QRadioButton" name="kcfg_VerboseLogging">
        <property name="toolTip">
         <string>Enable verbose debug output</string>
        </property>
        <property name="text">
         <string>&amp;Verbose</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">verbosityButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2" colspan="13">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="2" colspan="2">
       <widget class="QCheckBox" name="kcfg_INDIMountLogging">
        <property name="text">
         <string>Mount</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="6" colspan="2">
       <widget class="QRadioButton" name="kcfg_RegularLogging">
        <property name="toolTip">
         <string>Enable regular debug output</string>
        </property>
        <property name="text">
         <string>Reg&amp;ular</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">verbosityButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Verbosity:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="7">
       <widget class="QCheckBox" name="kcfg_INDIWeatherLogging">
        <property name="text">
         <string>Weather</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="10" column="1" rowspan="4">
       <widget class="QLabel" name="label_9">
        <property name="toolTip">
         <string>Have problems with INDI drivers? Turn on logging for drivers that exhibit issues.</string>
        </property>
        <property name="text">
         <string>Drivers</string>
        </property>
       </widget>
      </item>
      <item row="10" column="7" colspan="2">
       <widget class="QCheckBox" name="kcfg_INDIFocuserLogging">
        <property name="text">
         <string>Focuser</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="kcfg_INDILogging">
        <property name="toolTip">
         <string>Log INDI devices activity</string>
        </property>
        <property name="text">
         <string>INDI</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modulesGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QCheckBox" name="kcfg_INDIDomeLogging">
        <property name="text">
         <string>Dome</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="11" column="5">
       <widget class="QCheckBox" name="kcfg_INDIDetectorLogging">
        <property name="text">
         <string>Detector</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QCheckBox" name="kcfg_CaptureLogging">
        <property name="toolTip">
         <string>Log Ekos Capture module activity</string>
        </property>
        <property name="text">
         <string>Capture</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modulesGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QCheckBox" name="kcfg_FocusLogging">
        <property name="toolTip">
         <string>Log Ekos Focus module activity</string>
        </property>
        <property name="text">
         <string>Focus</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modulesGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="9">
       <widget class="QCheckBox" name="kcfg_SchedulerLogging">
        <property name="toolTip">
         <string>Log Ekos Scheduler module activity</string>
        </property>
        <property name="text">
         <string>Scheduler</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modulesGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="7" column="9">
       <widget class="QCheckBox" name="kcfg_FITSLogging">
        <property name="toolTip">
         <string>Log FITS processing activity</string>
        </property>
        <property name="text">
         <string>FITS</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modulesGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="10" column="5">
       <widget class="QCheckBox" name="kcfg_INDICCDLogging">
        <property name="text">
         <string>CCD</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="13" column="5">
       <widget class="QCheckBox" name="kcfg_INDIGPSLogging">
        <property name="text">
         <string>GPS</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QCheckBox" name="kcfg_INDIRotatorLogging">
        <property name="text">
         <string>Rotator</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="13" column="7">
       <widget class="QCheckBox" name="kcfg_INDIAOLogging">
        <property name="toolTip">
         <string>Adaptive Optics</string>
        </property>
        <property name="text">
         <string>AO</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driversGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="kcfg_showINDIMessages">
       <property name="toolTip">
        <string>Display INDI status messages in the status bar</string>
       </property>
       <property name="statusTip">
        <string>Display incoming INDI status messages in KStars status bar</string>
       </property>
       <property name="whatsThis">
        <string>Display incoming INDI status messages in KStars status bar</string>
       </property>
       <property name="text">
        <string>INDI messages in status &amp;bar</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="showLogsB">
       <property name="styleSheet">
        <string notr="true">font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Open Logs Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearLogsB">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Clear all logs</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="edit-delete">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../data/kstars.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="driversGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="verbosityButtonGroup"/>
  <buttongroup name="modulesGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
